<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpmStitchingGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 248, 248);</string>
    </property>
    <widget class="QListWidget" name="listWidget_files">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>351</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 0px;
border: 1px solid #999999;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;
font: 650 11pt &quot;Segoe UI&quot;;
background-color: rgb(248, 248, 248);</string>
     </property>
     <property name="text">
      <string>SPM Files Path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>431</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 248, 248);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_spm_type">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>240</y>
       <width>266</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #696969;
border-radius: 5px;</string>
     </property>
     <item>
      <property name="text">
       <string>Height Sensor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Height</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Height Trace</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Height Retrace</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;
font: 650 11pt &quot;Segoe UI&quot;;
background-color: rgb(248, 248, 248);</string>
     </property>
     <property name="text">
      <string>SPM Image Channel:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>listWidget_files</zorder>
    <zorder>label</zorder>
    <zorder>comboBox_spm_type</zorder>
    <zorder>label_7</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 248, 248);</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;
font: 650 11pt &quot;Segoe UI&quot;;
background-color: rgb(248, 248, 248);</string>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_preview_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>421</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 248, 248);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_3</zorder>
    <zorder>label_preview_image</zorder>
   </widget>
   <widget class="QWidget" name="widget_12" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>40</y>
      <width>51</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius: 0px;
border: 1px solid #999999;</string>
    </property>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 0px;
image: url(:/list/resource/Add.svg);
background:transparent
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 0px;
	image: url(:/list/resource/Sub.svg);
background:transparent
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_up">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 0px;
	image: url(:/list/resource/Up.svg);
background:transparent
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_down">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border: 0px;
	image: url(:/list/resource/Down.svg);
background:transparent
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_preview">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>475</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: qlineargradient(spread:pad, x1:0.511, y1:1, x2:0.517, y2:0, stop:0 rgba(221, 221, 221, 255), stop:1 rgba(240, 240, 240, 255));
 border: 1px solid #6E98D4;
font: 11pt &quot;Segoe UI&quot;;
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>475</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: qlineargradient(spread:pad, x1:0.511, y1:1, x2:0.517, y2:0, stop:0 rgba(221, 221, 221, 255), stop:1 rgba(240, 240, 240, 255));
 border: 1px solid #6E98D4;
font: 11pt &quot;Segoe UI&quot;;
}

QPushButton:hover
{
 border-radius : 4px;
 background-color: #DBE6F4;
 border: 1px solid #6E98D4;
}

QPushButton:pressed
{
 border-radius : 4px;
 background-color: #BBCEEA;
 border: 1px solid #3A73C2;

}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(248, 248, 248);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>5</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;
font: 650 11pt &quot;Segoe UI&quot;;
background-color: rgb(248, 248, 248);</string>
     </property>
     <property name="text">
      <string>Log</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_log">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>411</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);
border-radius: 0px;
border: 1px solid #696969;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 238);
border: 1px solid;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>label_2</zorder>
    <zorder>textBrowser_log</zorder>
   </widget>
   <zorder>widget_4</zorder>
   <zorder>widget_2</zorder>
   <zorder>widget_12</zorder>
   <zorder>widget_3</zorder>
   <zorder>btn_preview</zorder>
   <zorder>btn_save</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
